modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/META-INF/resources/admin/edit_application_left_column.jsp

```diff
@@ -29,7 +29,7 @@ OAuth2Application oAuth2Application = oAuth2AdminPortletDisplayContext.getOAuth2
 	</c:if>
 
 	<aui:select helpMessage="client-authentication-method-help" label="Client Authentication Method" name="clientAuthenticationMethod" required="<%= true %>">
-		<aui:option label="Client Secret Post" value="client_secret_post" />
```

Comment:
@ChrisKian  I think you are reliant on the dropdown having no explicit option selected when customers selected BASIC prior to this fix? That's a potential solution, but maybe risky if we add another new 1st option in  in the future? Did you consider an upgrade process?

Reply:
I've added an upgrade process in the new PR.  Thanks!